Form/Type/DocumentHiddenType.php

```diff
@@ -53,17 +54,16 @@ public function configureOptions(OptionsResolver $resolver)
                     'dm' => 'default',
                 ]
             )
-            ->setAllowedTypes(
```

Comment:
Symfony 3.* doesn't support that construction with an array inside. It should be split